javers-core/src/test/groovy/org/javers/core/diff/appenders/SetListChangeAppenderTest.groovy

```diff
@@ -0,0 +1,96 @@
+package org.javers.core.diff.appenders
+
+import org.javers.core.model.DummyUser
+import org.javers.core.model.SnapshotEntity
+import spock.lang.Unroll
+
+import java.time.LocalDate
+
+import static org.javers.core.diff.appenders.ContainerChangeAssert.getAssertThat
+import static org.javers.core.model.DummyUser.dummyUser
+
+/**
+ * @author Sergey Kobyshev
+ */
+class SetListChangeAppenderTest extends AbstractDiffAppendersTest {
```

Comment:
it's a copy of SetChangeAppenderTest.
Both classes should share the same tests but different `setupSpec` (see BeanBasedScannerTest, FieldBasedScannerTest)